docs/simplesamlphp-authproc.md

```diff
@@ -129,7 +129,6 @@ The following filters are included in the SimpleSAMLphp distribution:
 - [`core:AttributeAlter`](./core:authproc_attributealter): Do search-and-replace on attributevalues.
 - [`core:AttributeLimit`](./core:authproc_attributelimit): Limit the attributes in the response.
 - [`core:AttributeMap`](./core:authproc_attributemap): Change the name of the attributes.
-- [`core:AttributeRealm`](./core:authproc_attributerealm): (deprecated) Create an attribute with the realm of the user.
```

Comment:
For now, I would only mark the authproc as deprecated in the documentation, pointing to the alternative. Maybe add a log message if we don't have one. But I wouldn't remove it already.

Reply:
That all in place for years already. I've moved to milestone to 2.0

Reply:
Oh, jeez, I need coffee... 🤦‍♂ 

Sorry for the stupid comment Tim!

Reply:
I'd take some caffeine straight in the veins 🤣  No worries!